backend/app/Http/Controllers/ActivityController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Activity;
+use Auth;
+
+class ActivityController extends Controller
+{
+    public function show()
+{
+    $userId = Auth::id();
+    $activities = Activity::where('user_id', $userId)->get();
```

Comment:
Again, dont create a variable if you are going to use it one time
```suggestion
    $activities = Activity::where('user_id', Auth::id())->get();
```